core/src/main/java/io/aiven/klaw/service/UtilControllerService.java

```diff
@@ -69,11 +72,7 @@ public class UtilControllerService {
   @Value("${klaw.coral.enabled:false}")
   private boolean coralEnabled;
 
-  @Value("${klaw.sso.server.loginurl.azure}")
-  private String ssoServerLoginUrlAzure;
-
-  @Value("${klaw.sso.server.loginurl.google}")
-  private String ssoServerLoginUrlGoogle;
+  private Map<String, SSODetails> loginurl;
```

Comment:
```suggestion
  private Map<String, SSODetails> ssoProvidersLoginUrlsMap;
```

Reply:
so this is actually a component of @ConfigurationProperties(prefix = "klaw.sso.server") to be loginUrl I can change the application.properties 
Maybe I could change it to 'ssoProvider' from loginurl? but those need to map for it to map the details into SSODetails.